pkg/scheduler/framework/plugins/podtopologyspread/scoring_test.go

```diff
@@ -761,7 +761,7 @@ var (
 	}
 )
 
-func BenchmarkTestDefaultEvenPodsSpreadPriority(b *testing.B) {
+func BenchmarkTestDefaultPodTopologySpreadPriority(b *testing.B) {
```

Comment:
is this benchmark really useful given that we have a canonical set in integration tests?

Reply:
This can test against a specific function, and is faster than integration test, so I think it's still quite useful.